contacts/index.html

```diff
@@ -2,121 +2,164 @@
 <html>
   <head>
     <title>Chat Room</title>
+    <link rel="stylesheet" href="style.css">
   </head>
 
   <body>
       
     <header>
-      <h1>Chat Room</h1>
+      <h1 class="align-center">Chat Room</h1>
     </header>
 
     <nav>
-      <ul>
+      <ul class="align-center">
         <li><a href="..\index.html">Home</a></li>
+        <div class="separator">|</div>
         <li>Contacts</li>
       </ul>
     </nav>
       
     <main>
-      <h2>Contacts</h2>
-      <ul>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic1.jfif" alt="pic1" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Ionescu</span>
-            <span>Alin</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic2.jfif" alt="pic2" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Sherman</span>
-            <span>Leona</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic3.jfif" alt="pic3" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Carlson</span>
-            <span>Annette</span>
-            <button>X</button>
-          </div>
-        </li>
-        <li>
-          <div> 
-            <button>+</button>
-            <label>0</label>
-            <button>-</button>
-            <span><img src="images/star.png" alt="star icon" width="25" height="25"></span>
-            <span><img src="images/profile-pics/pic4.jfif" alt="pic4" width="25" height="25"></span>
-            <input type="checkbox">
-            <span>Watkins</span>
-            <span>Ervin</span>
-            <button>X</button>
+      
+      <div class="block">
+        <div class="contacts">
+          <h2 class="align-center type-title">Contacts</h2>
+
+          <ul class="list">
+            <li class="person-field">
+              <button>+</button>
+              <div class="number">0</div>
```

Comment:
Rename "number" class to "favs-count" or something similar. "Number" is too general.